GSA/made-in-america
# [FORMs-849]  MIA email hyperlink update

## Description

After reviewing the touchpoints survey some users were having issues with a hyperlink if their email client was not set up.  To help with this issue the hyperlink should who the email address for easygoing  access in case the user does not have this set up on their own

link to ticket: https://cm-jira.usa.gov/secure/RapidBoard.jspa?rapidView=3372&projectKey=FORMS&view=detail&selectedIssue=FORMS-849

## How to test

- pull down the branch 
- run `npm run start`
- go to browser and visit `localhost:4000`
- go to the contact page under customer service and check that the first bullet point has a hyperlink that is the OMB email address 

To prepare for code review, please follow the following checks.
